src/SignalR/clients/ts/FunctionalTests/SignalR.Client.FunctionalTestApp.csproj

```diff
@@ -34,6 +34,7 @@
     <Reference Include="Microsoft.AspNetCore.SignalR.Protocols.MessagePack" />
     <Reference Include="Microsoft.AspNetCore.SignalR" />
     <Reference Include="Microsoft.AspNetCore.StaticFiles" />
+    <Reference Include="Microsoft.Extensions.Hosting" />
```

Comment:
In the past we made Kestrel explicitly depend on Microsoft.AspNetCore.Hosting even though they only needed Hosting.Abstractions because we wanted to avoid everyone having to pull in both Microsoft.AspNetCore.Hosting and Kestrel. 

For some reason we didn't add the same direct relationship between AspNetCore.Hosting and Extensions.Hosting. Maybe we should do that to limit the upstream churn?